src/zcl_aff_writer.clas.testclasses.abap

```diff
@@ -187,7 +187,7 @@ CLASS ltcl_type_writer IMPLEMENTATION.
   ENDMETHOD.
 
   METHOD get_type_info_string_like_enum.
-    cl_abap_unit_assert=>assert_equals( exp = zif_aff_writer=>type_info-string act = cut->get_json_type_from_description( get_element_description( VALUE zif_aff_writer=>enum_type_info( ) ) ) ).
+    cl_abap_unit_assert=>assert_equals( exp = zif_aff_writer=>type_info-string act = cut->get_json_type_from_description( get_element_description( '' ) ) ).
```

Comment:
> yes, only "A"

In this case, we should revert this particular change. As far as I understand, this test shall verify, that the framework considers enums as `"type": "string"` in the JSON-schema.

Most probably, we have to define a local enum in the test and pass it to the method.

Reply:
Added my suggestion with https://github.com/SAP/abap-file-formats-tools/pull/159/commits/045a1e88b1623dc10b70d54e08ca801fd6757a72